spec/models/amendment_spec.rb

```diff
@@ -18,7 +18,7 @@
   end
 
   it "is invalid if its claim is not amendable" do
-    amendment = build(:amendment, claim: build(:claim, :rejected))
+    amendment = build(:amendment, claim: create(:claim, :rejected))
```

Comment:
I think this change becomes redundant if we move the factory to using `build` instead of `create`.